activist-org/activist
# style/add prefers reduced motion

### Contributor checklist




---

### Description
Users that have motion sickness and disable animations on their device, would expect not to see any animations on the platform. We can make use of the [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) feature which is widely supported, when applicable, we can use the [tailwind class](https://tailwindcss.com/docs/transition-property#prefers-reduced-motion) instead.
#### How to enable and test:
- Windows 10: Start > Settings > Ease of Access > Display > Toggle “Show Animations in Windows.”
- macOS: System Preferences > Accessibility > Display> Check “Reduce Motion.”
- iOS: Settings > Accessibility > Motion> Toggle “Reduce Motion.”
- Android: Settings > Accessibility > Toggle “Remove Animations.”


### Related issue





- #722
